packages/mantine/src/components/browser-preview/BrowserPreview.tsx

```diff
@@ -0,0 +1,70 @@
+import {InfoSize16Px} from '@coveord/plasma-react-icons';
+import {Box, ColorSwatch, Flex, Group, Stack, Text, Title, Tooltip, useMantineTheme} from '@mantine/core';
+import {PropsWithChildren} from 'react';
+
+export interface BrowserPreviewProps {
+    /**
+     * Text that will be displayed in the tooltip on the info icon
+     *
+     * @default The final look in your search page may differ due to the customization you made in your page.
+     *
+     */
+    headerTooltip?: string;
+    /**
+     * Custom title to be displayed at the center of the header.
+     */
+    title?: string;
+}
+
+export const BrowserPreview = ({
+    children,
+    headerTooltip = 'The final look in your search page may differ due to the customization you made in your page.',
+    title,
+}: PropsWithChildren<BrowserPreviewProps>) => {
+    const theme = useMantineTheme();
+    return (
+        <Stack
+            spacing={0}
+            maw={544}
+            mih={0}
+            style={{
+                boxShadow: theme.shadows.md,
+                borderRadius: theme.defaultRadius,
+                border: '1px solid',
+                borderColor: theme.colors.gray[3],
+                flex: 1,
+            }}
```

Comment:
Lets move this over to a BrowserPreview.styles.ts file using a createStyle function. You can follow this guide: https://v6.mantine.dev/guides/custom-components/